autoscheduler/frontend/src/redux/actions/courseCards.ts

```diff
@@ -36,57 +36,98 @@ function updateCourseCardSync(index: number, courseCard: CourseCardOptions): Upd
   };
 }
 
+/**
+ * Helper function that converts the given serialized section from the backend and converts it
+ * into a Section type
+ * @param sectionData The serialized section returned from the backend, i.e. from /api/sections
+ */
+function parseSection(sectionData: any): Section {
+  return new Section({
+    id: Number(sectionData.id),
+    crn: Number(sectionData.crn),
+    subject: sectionData.subject,
+    courseNum: sectionData.course_num,
+    sectionNum: sectionData.section_num,
+    minCredits: Number(sectionData.min_credits),
+    maxCredits: Number(sectionData.max_credits) || null,
+    currentEnrollment: Number(sectionData.current_enrollment),
+    maxEnrollment: Number(sectionData.max_enrollment),
+    honors: sectionData.honors,
+    web: sectionData.web,
+    instructor: new Instructor({ name: sectionData.instructor_name }),
+    grades: sectionData.grades == null ? null : new Grades(sectionData.grades),
+  });
+}
+
+/**
+ * Helper function that iterates through all of the meetings in sectionDAta and converts them
+ * into a Meeting's type array
+ * @param sectionData The serialized section returned from the backend, i.e. from /api/sections
+ * @param section The parsed Section type from this sectionData
+ */
+function parseMeetings(sectionData: any, section: Section): Meeting[] {
+  const meetings: Meeting[] = [];
+
+  sectionData.meetings.forEach((meetingData: any) => {
+    let start: string[] = ['0', '0'];
+    let end: string[] = ['0', '0'];
+
+    if (meetingData.start_time != null && meetingData.start_time.length > 0) {
+      start = meetingData.start_time.split(':');
+      end = meetingData.end_time.split(':');
+    }
+
+    const meeting = new Meeting({
+      id: Number(meetingData.id),
+      building: '',
+      meetingDays: meetingData.days,
+      startTimeHours: Number(start[0]),
+      startTimeMinutes: Number(start[1]),
+      endTimeHours: Number(end[0]),
+      endTimeMinutes: Number(end[1]),
+      // Converts string to MeetingType enum
+      meetingType: MeetingType[meetingData.type as keyof typeof MeetingType],
+      section,
+    });
+
+    meetings.push(meeting);
+  });
+
+  return meetings;
+}
+
+/**
+ * Helper function that parses all of the meetings in the section data and returns them.
```

Comment:
Based on the docstrings, `parseAllMeetings` and `parseMeetings` seem to do the same thing. Could you clarify that a bit?